index.html

```diff
@@ -3490,12 +3490,12 @@ <h4>JSON Web Token Extensions</h4>
               <li>
 <code>vc</code>: JSON object, which MUST be present in a JWT
 <a>verifiable credential</a>. The object contains the
-<a>verifiable credential</a> according to this specification.
```

Comment:
Can you clarify why a vast majority of the "verifiable" terms are being removed?

Reply:
probably because its not actually the same type... in JWT land, vc.issuanceDate is maybe not required if the 'instead of' path is taken... leading to type ambiguity if the term `verifiable credential` is used to refer to the `vc` payload... the jwt.payload.vc is in fact not verifiable... only the jwt (the outer container that includes it) make sit verifiable.

Reply:
this is in line with the langauge we use as well:

credential: JSON with an `@context` that contains all required fields except for `proof` -> issue -> verifiable credential `<JWT or LD Proof>`

Reply:
> if issuanceDate is a required attribute of a verifiable credential (thats what the spec says today)...then removing it destroys interoperability and conformance

Yes, and (removing `issuer` and `issuanceDate` and all other JWT-encoded properties) was exactly this was suggested on the call yesterday, which tells me that the entire VC-JWT ecosystem is not interoperable today.

Reply:
IMO, this change should be accepted, since `vc` is obviously not a `verifiable credential` under the current normative text... its possibly also not a `credential` under the current text, but this is a step in the right direction. We should not be standing in the way of a "more truthful" editorial refinement.